code/modules/vore/eating/bellymodes_datum_vr.dm

```diff
@@ -0,0 +1,247 @@
+GLOBAL_LIST_INIT(digest_modes, list())
+
+/datum/digest_mode
+	var/id = DM_HOLD
+	var/noise_chance = 0
+
+/datum/digest_mode/proc/process_mob(obj/belly/B, mob/living/L)
```

Comment:
It would be good to document the allowed return values for this proc.

Reply:
yeah true